packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -202,17 +204,13 @@ export const MessagesContainer = ( { currentUser }: ChatMessagesProps ) => {
 							);
 						} ) }
 						<JumpToRecent containerReference={ messagesContainerRef } />
-						{ chat.provider === 'odie' && (
-							<>
-								<ViewMostRecentOpenConversationNotice />
-								{ chat.status === 'dislike' && ! removeDislikeStatus && <DislikeThumb /> }
-								{ availableStatusWithFeedback.includes( chat.status ) && (
-									<div className="odie-chatbox__action-message">
-										{ chat.status === 'sending' && <ThinkingPlaceholder /> }
-										{ chat.status === 'dislike' && <DislikeFeedbackMessage /> }
-									</div>
-								) }
-							</>
+						<OdieNotices />
```

Comment:
Do we still need to add this condition: `chat.provider === 'odie'` below `<OdieNotices />`?

The View conversation notice appears for all conversations vs. production, where it only appears for AI conversations.

Reply:
Does the "View conversation" appear in prod even if I used `useViewMostRecentOpenConversationNotice( chat?.provider === 'odie' );`?